docs/source/quickguides/quick-guide-general.rst

```diff
@@ -67,16 +86,18 @@ Compy
 
 Cooley
 ^^^^^^
-<activation_path>: ``source /lus/theta-fs0/projects/ccsm/acme/tools/e3sm-unified/load_latest_e3sm_unified.sh``
+<activation_path>: ``source /lus/theta-fs0/projects/ccsm/acme/tools/e3sm-unified/load_latest_e3sm_unified_cooley.sh``
 
 <obs_path>:``/lus/theta-fs0/projects/ClimateEnergy_3/e3sm_diags/obs_for_e3sm_diags/``
 
 <test_data_path>:``/lus/theta-fs0/projects/ClimateEnergy_3/e3sm_diags/test_model_data_for_acme_diags/``
 
 
-Cori
-^^^^
-<activation_path>: ``source /global/cfs/cdirs/e3sm/software/anaconda_envs/load_latest_e3sm_unified.sh``
+Cori-Haswell
+^^^^^^^^^^^^
+[Note: there is known issue with running e3sm_diags on KNL, so running on Haswell is recommended.]
```

Comment:
@chengzhuzhang This line was manually added to the auto-generated Cori file in #489. I moved it here so it wouldn't be overwritten by the latest auto-generation. #509 will remove the auto-generated files to help clean up the repo.